src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -16,8 +16,8 @@ public class When_broadcasting_a_command : NServiceBusAcceptanceTest
         {
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<Sender>(b => b.When(c => c.EndpointsStarted, (session, c) => session.Send(new Request())))
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("1")); })
-                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.ScaleOut().InstanceDiscriminator("2")); })
+                .WithEndpoint<Receiver>(b => { b.CustomConfig(c => c.AddUniquelyAddressableQueue("1")); })
```

Comment:
> MakeInstanceUniquelyAddressable

I like that too, but I think the `Add` part plays a significant role here and in the usage scenario descibed by @SeanFeldman (otherwise they may think the queue name is just changed).

@danielmarbach definitely has a good point, but it seems to be a tradeoff to me, where I think that the fact that we add an additional queue is more important than the naming of the queue?